test/DurableTaskExtensionMock.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
```

Comment:
All source files need to include the copyright header.

Reply:
I will add it shortly.